test/controllers/management/helpers.test.js

```diff
@@ -0,0 +1,123 @@
+const { createPoliciesBody } = require('../../../controllers/management/helpers');
+const moment = require('moment');
+const sinon = require('sinon');
+const { expect } = require('chai');
+
+describe('Management controller helpers tests: ', () => {
+	describe("createPoliciesBody", () => {
+		let momentFormatMock;
+		
+		beforeEach(() => {
+			momentFormatMock = sinon.mock(moment.fn);
+		});
+
+		afterEach(() => {
+			momentFormatMock.restore();
+		});
+
+		it("should call moments format method with proper string format", () => {
+			// given
+			const policiesData = [{}];
+			momentFormatMock.expects('format').withArgs('DD.MM.YYYY HH:mm').once();
+			// when
+			createPoliciesBody(policiesData);
+			// then
+			momentFormatMock.verify();
+		});
+
+		it("should not loose any consent policy", () => {
```

Comment:
'lose' - does it mean nothing should be deleted? why are you comparing the constant to itelf? Did I miss anything?

Reply:
it just checks if this function make a 1 to 1 transform without any data loosing

Reply:
okay, makes sense